facefusion/face_analyser.py

```diff
@@ -534,9 +543,23 @@ def get_many_faces(vision_frames : List[VisionFrame]) -> List[Face]:
 			if static_faces:
 				faces = static_faces
 			else:
-				bounding_boxes, face_landmarks_5, face_scores = detect_faces(vision_frame)
-				if bounding_boxes and face_landmarks_5 and face_scores and facefusion.globals.face_detector_score > 0:
-					faces = create_faces(vision_frame, bounding_boxes, face_landmarks_5, face_scores)
+				bounding_boxes_all = []
+				rotated_bounding_boxes_all = []
+				face_landmarks_5_all = []
+				face_scores_all = []
+
+				for angle in facefusion.globals.face_detector_angles:
+					if angle == 0:
+						bounding_boxes, face_landmarks_5, face_scores = detect_faces(vision_frame)
+					else:
+						bounding_boxes, face_landmarks_5, face_scores = detect_faces_with_rotation(vision_frame, angle)
+					rotated_bounding_boxes = [ convert_to_rotated_bounding_box(bounding_box, angle) for bounding_box in bounding_boxes ]
```

Comment:
I do not fully understand why convert_to_rotated_bounding_box() is needed as you pass rotated_bounding_boxes already to the method?